src/components/Timeline/TimelineAddNote.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import TextEditor from './TextEditor';
+
+const TimelineAddNote: React.FunctionComponent = () => {
+  return (
+    <TextEditor
+      initialValue={[
+        {
+          children: [{ text: 'Add a new note' }],
+          type: 'paragraph',
+        },
+      ]}
+    />
```

Comment:
Instead of initial value, what happens if we use a placeholder? If it works as I'd expect, I think that would be better. That way, focusing the textfield for the first time starts with an empty field (where you don't have to start by deleting the initial value).